components/rhs_root_post/rhs_root_post.jsx

```diff
@@ -342,7 +343,7 @@ class RhsRootPost extends React.PureComponent {
         if (!isPostDeleted && this.props.post.type !== Constants.PostTypes.FAKE_PARENT_DELETED) {
             dotMenuContainer = (
                 <div
-                    ref='dotMenu'
+                    ref={this.dotMenuRef}
```

Comment:
I don't think this is defined? It think you need either `this.getDotMenuRef` or `this.dotMenu` here.

Reply:
Yes, I think you right. I will fix it.